src/main/java/Level6.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.*;
+public class Level6 {
+    public static void main(String[] args){
+        String input;
+        ArrayList<Task> task = new ArrayList<Task>();
+        Scanner in = new Scanner(System.in);
+        System.out.println("____________________________________________________________\n");
+        System.out.println(" Hello! I'm Duke\n");
+        System.out.println(" What can I do for you?\n");
+        System.out.println("____________________________________________________________\n");
+        input=in.nextLine();
+        while(!input.equals("bye")){
+            try {
+                if (input.equals("list")) {
+                    System.out.println("____________________________________________________________\n");
+                    for (int i = 0; i < task.size(); i++) {
+
+                        System.out.println((i + 1) + ":" + "[" + task.get(i).getStatusIcon() + "] " + task.get(i).description);
+                    }
+                    System.out.println("____________________________________________________________\n");
+                } else if (input.startsWith("done")) {
+                    System.out.println("____________________________________________________________\n");
+                    int taskNumberCompleted = Integer.parseInt(input.replaceAll("\\D+", "")) - 1;
+                    task.set(taskNumberCompleted, task.get(taskNumberCompleted).completeTask());
+                    System.out.println(" Nice! I've marked this task as done:");
+                    System.out.println(" " + ":" + "[" + task.get(taskNumberCompleted).getStatusIcon() + "] " + task.get(taskNumberCompleted).description);
+                    System.out.println("____________________________________________________________\n");
+                } else if (input.startsWith("todo")) {
+                    Todo todo = new Todo(input);
+                    task.add(todo);
+                    System.out.println("____________________________________________________________\n");
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println("   " + todo.toString());
+
+                    System.out.println("Now you have " + task.size() + " todo borrow booktasks in the list. ");
+                    System.out.println("____________________________________________________________\n");
+
+
+
+                }
+                else if (input.startsWith("delete")) {
+                    System.out.println("____________________________________________________________\n");
+                    System.out.println("Noted. I've removed this task:  ");
+                    int taskNumberdeleted = Integer.parseInt(input.replaceAll("\\D+", "")) - 1;
```

Comment:
Name can be better changed to `deletedTaskNumber`. Keep in mind that variable name convention is camelCase, and it is better to write names like how they are said in normal conversation.